### DISCUSSION TITLE
Problem:  Stumpwm + Sketch + Lem = Problem

### DISCUSSION THREAD

**Discussion by aykaramba** [lem-project/lem#1315] [General]
For clarification:  

1) I don't run any desktop environment.
2) i just run the Stumpwm window manager.
3) I run an AMD gpu with fully supported drivers, nothing exotic, old card fully supported.
4) All 3d accelerated application such as Gimp and games work perfectly on the environment.

Problem:  Lem-SDL2 works great on Stumpwm, no issues.  However, when I startup the Sketch (https://github.com/vydd/sketch) package and start an instance of one of the examples or any new window it crashes Lem.  Sketch uses cl-sdl2 so I assume there is some interaction here that causes Lem to hang.  Basically, on Stumpwm Lem rendering of the window stops and while I can cycle through Lem / Sketch windows nothing renders.  This could be something to do with Stumpwm + Lem + Sketch as well and I need to test that.

Question: Anyone here running Stumpwm?  Are you able to load Sketch examples from the Lem lisp repl?

**Comment by cxxxr** [comment]
If you are running lem and sketch on the same sbcl process, crashing is, well, likely.
How about a new M-x slime on lem and use sketch there?

**Comment by aykaramba** [comment]
Oh, that works!  Okay, I learned something new.  I have more to learn about Lisp and SBCL processes.  Thanks.
